spec/locales-currencies-tz.html

```diff
@@ -189,30 +200,33 @@ <h1>IsValidTimeZoneName ( _timeZone_ )</h1>
       <h1>
         CanonicalizeTimeZoneName (
           _timeZone_: a String value that is a valid time zone name as verified by IsValidTimeZoneName,
-        )
+        ): a String
       </h1>
       <dl class="header">
         <dt>description</dt>
         <dd>It returns the canonical and case-regularized form of _timeZone_.</dd>
       </dl>
       <emu-alg>
         1. Let _ianaTimeZone_ be the String value of the Zone or Link name of the IANA Time Zone Database that is an ASCII-case-insensitive match for _timeZone_.
-        1. If _ianaTimeZone_ is a Link name, let _ianaTimeZone_ be the String value of the corresponding Zone name as specified in the file <code>backward</code> of the IANA Time Zone Database.
+        1. If _ianaTimeZone_ is a Link name, set _ianaTimeZone_ to the String value of the corresponding Zone name as specified in the file <code>backward</code> of the IANA Time Zone Database.
         1. If _ianaTimeZone_ is one of *"Etc/UTC"*, *"Etc/GMT"*, or *"GMT"*, return *"UTC"*.
         1. Return _ianaTimeZone_.
       </emu-alg>
     </emu-clause>
 
     <emu-clause id="sup-defaulttimezone" oldids="sec-defaulttimezone" type="implementation-defined abstract operation">
-      <h1>DefaultTimeZone ( ): a String</h1>
+      <h1>
+        DefaultTimeZone (
```

Comment:
```suggestion
        SystemTimeZoneIdentifier (
```

Reply:
I'm fine with leaving this, since Temporal will remove it from ECMA-402 anyway (and it might even be possible to remove before then, now that https://github.com/tc39/ecma402/pull/788 has landed).

Reply:
If we're making changes anyway, why not align 402 to the current 262 naming?  Is there any downside to doing so?

Reply:
#788 removed the `DefaultTimeZone` operation, so this change is no longer needed.